sklearn/multioutput.py

```diff
@@ -483,6 +483,7 @@ class ClassifierChain(_BaseChain, ClassifierMixin, MetaEstimatorMixin):
     of models that are earlier in the chain.
 
     Read more in the :ref:`User Guide <classifierchain>`.
+    See Also :ref:`Regressor Chain <regressorchain>`.
```

Comment:
like this: https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/tree/tree.py#L1029

Reply:
Sorry, I was not very clear with what exactly is referred by 'docstring'. Changes added.